hypervisor/debug/shell_priv.h

```diff
@@ -36,62 +36,64 @@ struct shell {
 
 #define SHELL_CMD_DUMPMEM		"dumpmem"
 #define SHELL_CMD_DUMPMEM_PARAM		"<addr, length>"
-#define SHELL_CMD_DUMPMEM_HELP		"Dump physical memory"
+#define SHELL_CMD_DUMPMEM_HELP		"Dump host memory, starting at a given address, and for a given length (in "\
+					"bytes)"
 
 #define SHELL_CMD_SOS_CONSOLE		"sos_console"
 #define SHELL_CMD_SOS_CONSOLE_PARAM	NULL
-#define SHELL_CMD_SOS_CONSOLE_HELP	"Switch to SOS's console"
+#define SHELL_CMD_SOS_CONSOLE_HELP	"Switch to the SOS's console. Use [Ctrl+Spacebar] to return to the ACRN shell "\
+					"console"
 
 #define SHELL_CMD_INTERRUPT		"int"
 #define SHELL_CMD_INTERRUPT_PARAM	NULL
-#define SHELL_CMD_INTERRUPT_HELP	"show interrupt info per CPU"
+#define SHELL_CMD_INTERRUPT_HELP	"List interrupt information per CPU"
 
 #define SHELL_CMD_PTDEV			"pt"
 #define SHELL_CMD_PTDEV_PARAM		NULL
-#define SHELL_CMD_PTDEV_HELP		"show pass-through device info"
+#define SHELL_CMD_PTDEV_HELP		"Show pass-through device information"
 
 #define SHELL_CMD_REBOOT		"reboot"
 #define SHELL_CMD_REBOOT_PARAM		NULL
-#define SHELL_CMD_REBOOT_HELP		"trigger system reboot"
+#define SHELL_CMD_REBOOT_HELP		"Trigger a system reboot (immediately)"
 
 #define SHELL_CMD_IOAPIC		"dump_ioapic"
 #define SHELL_CMD_IOAPIC_PARAM		NULL
-#define SHELL_CMD_IOAPIC_HELP		"show native ioapic info"
+#define SHELL_CMD_IOAPIC_HELP		"Show native IOAPIC information"
 
 #define SHELL_CMD_VIOAPIC		"vioapic"
 #define SHELL_CMD_VIOAPIC_PARAM		"<vm id>"
-#define SHELL_CMD_VIOAPIC_HELP		"show vioapic info"
+#define SHELL_CMD_VIOAPIC_HELP		"Show virtual IOAPIC (vIOAPIC) information for a specific VM"
 
 #define SHELL_CMD_LOG_LVL		"loglevel"
-#define SHELL_CMD_LOG_LVL_PARAM		"[<console_loglevel> [<mem_loglevel> " \
-					"[npk_loglevel]]]"
-#define SHELL_CMD_LOG_LVL_HELP		"get(para is NULL), or set loglevel [0-6]"
+#define SHELL_CMD_LOG_LVL_PARAM		"[<console_loglevel> [<mem_loglevel> [npk_loglevel]]]"
+#define SHELL_CMD_LOG_LVL_HELP		"No argument: get the level of logging for the console, memory and npk. Set "\
+					"the level by giving (up to) 3 parameters between 0 and 6 (verbose)"
 
 #define SHELL_CMD_CPUID			"cpuid"
 #define SHELL_CMD_CPUID_PARAM		"<leaf> [subleaf]"
-#define SHELL_CMD_CPUID_HELP		"cpuid leaf [subleaf], in hexadecimal"
+#define SHELL_CMD_CPUID_HELP		"Display the CPUID leaf [subleaf], in hexadecimal"
 
 #define SHELL_CMD_RDMSR			"rdmsr"
 #define SHELL_CMD_RDMSR_PARAM		"[-p<pcpu_id>]	<msr_index>"
-#define SHELL_CMD_RDMSR_HELP		"rdmsr -p<pcpu_id> <msr_index>, msr_index in hexadecimal"
+#define SHELL_CMD_RDMSR_HELP		"Read the MSR at msr_index (in hexadecimal) for CPU ID pcpu_id"
 
 #define SHELL_CMD_WRMSR			"wrmsr"
 #define SHELL_CMD_WRMSR_PARAM		"[-p<pcpu_id>]	<msr_index> <value>"
-#define SHELL_CMD_WRMSR_HELP		"wrmsr -p<pcpu_id> <msr_index> <value>, msr_index and value in hexadecimal"
+#define SHELL_CMD_WRMSR_HELP		"Write value (in hex) to the MSR at msr_index (in hex) for CPU ID pcpu_id"
```

Comment:
It would be better to use "hexadecimal" consistently.

Reply:
I agree in principle, but I was trying to save some text/characters as this is displayed over a serial line connection and too much text tends to clutter the output a bit. I can change it though :-)

Reply:
Just force-pushed a new version with `hexadecimal` fully spelled out.